app/src/main/res/values-nl/strings.xml

```diff
@@ -71,11 +71,11 @@
     <string name="invalid_image_path">Ongeldig pad naar afbeelding</string>
     <string name="image_editing_failed">Fout bij bewerken van afbeelding</string>
     <string name="edit_image_with">Afbeelding bewerken met:</string>
-    <string name="no_editor_found">Geen app gevonden voor bewerking</string>
+    <string name="no_editor_found">Geen bewerk-app gevonden</string>
     <string name="unknown_file_location">Onbekende bestandslocatie</string>
-    <string name="error_saving_file">Kan het bronbestand niet overschrijven</string>
-    <string name="rotate_left">Linksom draaien</string>
-    <string name="rotate_right">Rechtsom draaien</string>
+    <string name="error_saving_file">Het bronbestand kan niet worden overschreven</string>
```

Comment:
I think this is a toast message, so it disappears after a short while, so keeping it more concise would be better.

Reply:
Yes, but the way you translated it is the Flemish way.

Reply:
There's nothing Flemish to it. I'm Dutch.

Reply:
In that case you may want to review a few things, 'cause you do use a few Flemish expressions.

Reply:
There's nothing Flemish about it, it's just concise. But if the sentence is not cut off by the toast, the toast msg doesn't cover a huge part of the screen, and it's presented long enough to actually be read, then it's fine.

Reply:
My Flemish colleagues say it's Flemish, but whatever. It's not cut off by the toast.